src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_drawing_2014_chartex.g.cs

```diff
@@ -2174,22 +2375,22 @@ public Xsddouble Xsddouble
         }
 
         /// <summary>
-        /// <para>BinCountXsdunsignedInt.</para>
+        /// <para>XsdunsignedInt.</para>
         /// <para>Represents the following element tag in the schema: cx:binCount.</para>
         /// </summary>
         /// <remark>
         /// xmlns:cx = http://schemas.microsoft.com/office/drawing/2014/chartex
         /// </remark>
-        public BinCountXsdunsignedInt BinCountXsdunsignedInt
+        public XsdunsignedInt XsdunsignedInt
```

Comment:
This will be a breaking change which is not okay. There's a way around it, but let's figure out why this is changed.

Reply:
fixed